ansible/playbooks/nagios/roles/Nagios_Config/scripts/nagios_yaml.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+
+import argparse
+import json
+import os
+import subprocess
+import sys
+from os import path
+
+import yaml
+try:
+    import configparser
+except ImportError:
+    import ConfigParser as configparser
+
+valid_types = ('build', 'dockerhost', 'test')
+
+Output_Path = sys.argv[1]
+
+def main():
+
+    # load config file for special cases
+    config = configparser.RawConfigParser()
+    config.read('ansible.cfg')
+
+    # load public inventory
+    basepath = path.dirname(__file__)
+    inventory_path = "/tmp/ansible_inventory.yml"
```

Comment:
set argv[2] for `inventory_path`
do not hardcoded to it. otherwise, this py wont be run unless it is right after `Download inventory file` is done

Reply:
@steelhead31 suggested keep it 1 , any further pointers why?

Reply:
in this case,  you have to bind both "download" and "running python script" in one go.
i.e if you opt-out "download" it might not work for the "python" when it is a new server without  "/tmp/ansible_inventory.yml"
if you have another input parameter to python script, you can be more flex to set the inventory file.

Reply:
Going to note it , at test out after the recent commit

Reply:
A first walk around for now  would be to add a pre task to check ,
if the inventory doesnt exist, or has been skipped then it wont attempt to run the python